ia/src/parser.py

```
@@ -0,0 +1,41 @@
+
+        if not isinstance(machine, str) or machine.strip() == "":
+            exit_error("Error: Invalid machine hostname (-h).")
+        Parser = iaParser(
+            port=port,
+            name=name,
+            machine=machine
+        )
+        return Parser
+    except ValueError as e:
+        exit_error(f"Error parsing: {e}")
+        return None
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)